packages/nx-plugin-openapi/src/executors/generate-api-lib-sources/executor.spec.ts

```diff
@@ -29,18 +32,28 @@ xdescribe('Command Runner Builder', () => {
         projects: {
           proj: {
             root: '',
-            sourceRoot: 'src',
+            sourceRoot: `./tmp/src/${dir}`,
             targets: {},
           },
         },
       },
       isVerbose: false,
-    };
+    });
   });
 
   it('can run', async () => {
-    const { success } = await executor(schema, context);
+    jest.setTimeout(30_000);
+    const { success } = await executor(schema, context('local'));
+    // Expect that it succeeded.
+    expect(success).toBe(true);
+    expect(existsSync(`./tmp/src/local/index.ts`)).toBe(true);
```

Comment:
🤔  I don't know about this test: it looks pretty much like a e2e test.

What if we just don't unit test this executor? Lots of executors by Nrwl don't have unit tests, but I've seen some e2e tests.

How does it sound to you to delete this and move its logic into a dedicated spec file inside `packages/nx-plugin-openapi-e2e/tests`?